http-agent/src/main/resources/webapp/messages/messages.properties

```diff
@@ -0,0 +1,7 @@
+http.agent=HTTP-Agent
+http-agent.request=Request
+http-agent.url=URL
+http-agent.username=Username
+http-agent.password=Password
+http-agent.followRedirects=Follow Redirects
+http-agent.bodyParameters=Request Body
```

Comment:
Either rename the key to "http-agent.requestBody" or the value to "Body Parameters".